src/main/java/thePackmaster/cards/showmanpack/Showstopper.java

```diff
@@ -0,0 +1,31 @@
+package thePackmaster.cards.showmanpack;
+
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.showmanpack.ShowstopperAction;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Showstopper extends AbstractShowmanCard {
+    public final static String ID = makeID("Showstopper");
+    private final static int BLOCKTHRESHOLD = 100;
+    private final static int DAMAGE = 50;
+
+    public Showstopper() {
+        super(ID, -1, CardType.SKILL, CardRarity.RARE, CardTarget.SELF);
+        baseMagicNumber = magicNumber = 8;
+        baseDamage = damage = DAMAGE;
+        this.exhaust = true;
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        int energy = this.energyOnUse;
+        if (upgraded){
```

Comment:
Since it's simple, it'd be nice if you add the timesUpgraded variable instead of doing it like this, taht way it's compatible with pande's force upgrading in blackmsith pack

Reply:
Good idea